.pre-commit-config.yaml

```diff
@@ -1,11 +1,16 @@
-exclude: 'tests/test_data/'
+exclude: '.testhtml'
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.4.0
     hooks:
     -   id: check-yaml
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
+    -   id: pretty-format-json
```

Comment:
Ah; this is [available built-in to `pre-commit`](https://github.com/pre-commit/pre-commit-hooks/blob/7b2234e6dd211438f6416cd261fa2e747ef498cf/pre_commit_hooks/pretty_format_json.py) already?  Neat.